sources/packages/web/src/views/institution/WithdrawalUpload.vue

```diff
@@ -0,0 +1,316 @@
+                    </li>
+                  </ul>
+                </div>
+              </template></Column
+            >
+          </DataTable>
+        </content-group>
+      </content-group>
+    </content-group>
+  </full-page-container>
+</template>
+
+<script lang="ts">
+import { ref, computed, defineComponent } from "vue";
+import {
+  DEFAULT_PAGE_LIMIT,
+  PAGINATION_LIST,
+  BannerTypes,
+  VForm,
+  InputFile,
+  ApiProcessError,
+} from "@/types";
+import { useFormatters, useSnackBar } from "@/composables";
+import { FileUploadProgressEventArgs } from "@/services/http/common/FileUploadProgressEvent";
+import { ApplicationService } from "@/services/ApplicationService";
+import { ApplicationBulkWithdrawal } from "@/types/contracts/institution/Application";
+import { APPLICATION_WITHDRAWAL_INVALID_TEXT_FILE_ERROR } from "@/constants";
+
+const ACCEPTED_FILE_TYPE = "text/plain";
+const MAX_APPLICATION_WITHDRAWAL_UPLOAD_SIZE = 4194304;
+
+export default defineComponent({
+  setup() {
+    const snackBar = useSnackBar();
+    const validationProcessing = ref(false);
+    const creationProcessing = ref(false);
+    const { dateOnlyLongString } = useFormatters();
+    // Only one will be used but the component allows multiple.
+    const withdrawalFiles = ref<InputFile[]>([]);
+    // Possible errors and warnings received upon file upload.
+    const validationResults = ref([] as ApplicationBulkWithdrawal[]);
+    const uploadForm = ref({} as VForm);
+    const uploadProgress = ref({} as FileUploadProgressEventArgs);
+    // Workaround to reset the file upload component to its original state.
+    // It is apparently a vuetify beta issue. It can be removed once there is a
+    // better way to force the component to reset its state.
+    const textFileUploadKey = ref(0);
+    // Specific error message to be display the error that occurs when the file upload
+    // has a file selected and the user changes its contents (net::ERR_UPLOAD_FILE_CHANGED).
+    const showPossibleFileChangeError = ref(false);
+
+    const uploadFile = async (validationOnly: boolean) => {
+      const validationResult = await uploadForm.value.validate();
+      if (!validationResult.valid) {
+        return;
+      }
+      validationResults.value = [];
+      showPossibleFileChangeError.value = false;
+      try {
+        if (validationOnly) {
+          validationProcessing.value = true;
+        } else {
+          creationProcessing.value = true;
+        }
+        const [fileToUpload] = withdrawalFiles.value;
+        const uploadResults =
+          await ApplicationService.shared.applicationBulkWithdrawal(
+            fileToUpload,
+            validationOnly,
+            (progressEvent: FileUploadProgressEventArgs) => {
+              uploadProgress.value = progressEvent;
+            },
+          );
+        validationResults.value = uploadResults;
+        if (uploadResults.length) {
+          validationResults.value = uploadResults;
```

Comment:
with line 239, do we need 241 ? Let me know the intention , if I am missing anything

Reply:
THe first if is to check if the uploadResults have some value and dummy assignment, to display the snackbar in the else if.

Reply:
I believe `validationResults.value = uploadResults;` is only needed for `if (uploadResults.length) {`, in the rest of the case it empty array, which is already assigned in line 222 right?